scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -136,11 +136,13 @@ private object RunnerContext {
     pipelineOptions: PipelineOptions,
     classLoader: ClassLoader
   ): Iterable[String] = {
-    val matchesEnvDir: String => Boolean = _.matches(s"${sys.props("user.home")}/\\..+/.+")
+    val sanitize: String => String = _.replace("\\", "/")
+    val matchesEnvDir: String => Boolean =
+      _.matches(s"${sanitize(sys.props("user.home"))}/\\..+/.+")
```

Comment:
since L145 sanitize before call matcheEvnDir, is sanitize redundant here?

Reply:
Whoops fat fingered & merged. Feel free to amend later.

Reply:
No, that it's not redundant. That is sanitising the `sys.prop`